apps/fluent-tester/src/TestComponents/Checkbox/CheckboxTest.tsx

```diff
@@ -165,24 +189,39 @@ const checkboxSections: TestSection[] = [
     component: TokenCheckbox,
   },
   {
-    name: 'Checkbox for E2E Testing',
-    component: E2ECheckboxTest,
+    name: 'E2E Testing for CheckboxV1',
+    component: E2ECheckboxV1Test,
+  },
+  {
+    name: 'Legacy - Basic Checkboxes',
+    testID: CHECKBOX_TESTPAGE,
+    component: BasicCheckbox_legacy,
+  },
+  {
+    name: 'Legacy - Other Implementations',
+    component: OtherCheckbox_legacy,
+  },
+  {
+    name: 'Legacy - Token Customized Checkboxes',
+    component: TokenCheckbox_legacy,
+  },
```

Comment:
Is the idea that as we go forward, Legacy test pages will be combined into the V1 test pages at the bottom for other controls too? Or is checkbox special?

Reply:
OK, thanks. Just to confirm:

The northstar looks something like: 
- (already done) - Button Legacy is at the bottom of ButtonV1
- Tabs Legacy will go to bottom of tab
- MenuButtonLegacy is at the bottom of MenuButton
and...
- Maybe not move stuff like Persona into Avatar or ContextualMenu into Menu, since those are quite different / have different names?

Reply:
Yeah, that's the plan